package.json

```diff
@@ -21,9 +22,12 @@
     "prop-types": "15.7.2",
     "react": "16.14.0",
     "react-dom": "16.14.0",
+    "start-server-and-test": "1.11.7",
     "swr": "^0.4.0"
   },
   "devDependencies": {
+    "@netlify/plugin-nextjs": "1.1.0",
```

Comment:
Curious why this has appeared in this pr 🤔 didn't this dep go in another branch?

Reply:
I think it did, but probably got added when I checked out from main and pulled this branch down to merge the `package.json` files and fix the merge conflicts in the `yarn.lock`